gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# 75 interested going

### Description

Made click events on going and interested for events mutually exclusive.

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
```

1.  Click on any event and mark it as interested
2. Mark the same events as going and check the home or profile tab to see if that event is moved from interested stack to going stack.
3. Do the same for an event marked going and change to interested.
4. Check if the color of the interested/going button change to red and back to teal when we select/unselect going/interested options respectively.

### Miscellaneous Comments
None

### Checklist
